### ISSUE TITLE
scroll not working in mobile browser

### ISSUE THREAD

**Issue by Abdul-aziz-shuvo** [ankeetmaini/react-infinite-scroll-component#303]
working on desktop fine..but not working in mobile browser

**Comment by ianujvarshney** [comment]
yes i did use on desktop mode its worked but when i will use on mobile view it shows by default just three post.

**Comment by fbudassi** [comment]
I can confirm this, it's still happening, version 6.1.0.
